portfolio/src/main/webapp/script.js

```diff
@@ -19,13 +19,35 @@ function reviewsPage()
     window.location.href = "reviews.html";
 }
 
-/* gets a random welcome message from /data serverlet and displays it in the toptext */
+/* gets a random welcome message from /welcome serverlet and displays it in the toptext */
 async function getRandomMessage() {
-  const response = await fetch('/data');
+  const response = await fetch('/welcome');
   const message = await response.text();
   document.getElementById('top-text').innerText = message;
 }
 
+/* gets the reviews from /data servlet */
+async function getReviewData() {
+  const response = await fetch('/data');
+  const message = await response.json();
+
+  for(var i = 0; i < message.length; i++) {
+    document.getElementById('white-background').innerHTML += "<div id='review-box'> <p id='review-text'>" + message[i] + "</p></div>";
+  }
+
+  /* alternate attempt which didn't work: 
+  *
+  reviewBox = document.getElementById('white-background');
+  for(var i = 0; i < message.length; i++) {
+    newComment = document.createElement('review-text');
+    newComment.innerText = message[i];
+    newBox = document.createElement('review-box');
+    newBox.appendChild(newComment);
+    reviewBox.appendChild(newBox);
+  } */
```

Comment:
Tried to execute this code in a cleaner manner (the commented code) but could not get it to work. Is there a better way to do this?

Reply:
Sure! Could you try this: https://jsfiddle.net/8r7Lpwtm/7/

```
reviewBox = document.getElementById('white-background');
  for(var i = 0; i < message.length; i++) {
    // `createElement` takes in the tag.
    newComment = document.createElement('div');
    // Use `setAttribute` to set the id.
    newComment.setAttribute('id', 'review-text');
    newComment.innerText = message[i];
    newBox = document.createElement('p');
    newBox.setAttribute('id', 'review-box');
    newBox.appendChild(newComment);
    reviewBox.appendChild(newBox);
  }
```

Reply:
oh I get it now! thank you